kgitb/__main__.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import sys
+
+from flask import Flask
+
+from .config import GH_TOKEN, HOST, PORT, HIDDEN
+from . import routes
+
+LOG = logging.getLogger(__name__)
+
+
+def main():
+    app = routes.setup(Flask(HIDDEN[1:-1]))
+    app.run(host=HOST, port=PORT)
+
+
+if __name__ == '__main__':
+    if GH_TOKEN is None:
+        sys.exit("GH_TOKEN must be set.")
+    LOG.basicConfig(level=logging.DEBUG)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Instance of 'RootLogger' has no 'basicConfig' member [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=Pylint%3AE1101)